sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubListenerTests.cs

```diff
@@ -379,6 +379,142 @@ public async Task ProcessEvents_Failure_Checkpoints()
                 Times.Once);
         }
 
+        /// <summary>
+        /// If the partition processor token is signaled, we should NOT checkpoint as the partition ownership
```

Comment:
Even though we no longer have code around the stopping reason and execution result, I left the various test cases in case someone tries to add it in the future.